src/main.cpp

```diff
@@ -1590,6 +1590,18 @@ static void AlertNotify(const std::string& strMessage, bool fThread)
         runCommand(strCmd);
 }
 
+static void AlertNotifyOnce(const std::string& strMessage, bool fThread)
```

Comment:
This was a mistake, fixed.

First I though about removing the `fThread` argument completely (because it really is `true` everywhere, but then decided to keep this small). What do you think?